src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -0,0 +1,286 @@
+        headerMap.put("manageOrderHeader1", getHeaderInfo(caseData));
+        return headerMap;
+    }
+
+    public CaseData getUpdatedCaseData(CaseData caseData) {
+        return CaseData.builder().childrenList(getChildInfoFromCaseData(caseData))
+            .selectedOrder(getSelectedOrderInfo(caseData)).build();
+    }
+
+    private Map<String,String> getOrderTemplateAndFile(CreateSelectOrderOptionsEnum selectedOrder) {
+        Map<String,String> fieldsMap = new HashMap();
+        switch (selectedOrder) {
+            case blankOrderOrDirections:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c21TDraftTemplate);
+                fieldsMap.put(PrlAppsConstants.FILE_NAME, c21DraftFile);
+                fieldsMap.put(PrlAppsConstants.FINAL_TEMPLATE_NAME, c21Template);
+                fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c21File);
+                break;
+            case standardDirectionsOrder:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE,"");
+                fieldsMap.put(PrlAppsConstants.FILE_NAME, "");
+                break;
+            case childArrangementsSpecificProhibitedOrder:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c43DraftTemplate);
+                fieldsMap.put(PrlAppsConstants.FILE_NAME, c43DraftFile);
+                fieldsMap.put(PrlAppsConstants.FINAL_TEMPLATE_NAME, c43Template);
+                fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c43File);
+                break;
+            case specialGuardianShip:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c43ADraftTemplate);
+                fieldsMap.put(PrlAppsConstants.FILE_NAME, c43ADraftFilename);
+                break;
+            default:
+                break;
+        }
+        return fieldsMap;
+    }
+
+    private String getSelectedOrderInfo(CaseData caseData) {
+        StringBuilder selectedOrder = new StringBuilder();
+        selectedOrder.append(caseData.getManageOrdersOptions() == ManageOrdersOptionsEnum.createAnOrder
+                                 ? caseData.getCreateSelectOrderOptions().getDisplayedValue()
+                                 : caseData.getChildArrangementOrders().getDisplayedValue());
+        selectedOrder.append("\n\n");
+        return selectedOrder.toString();
+    }
+
+    private String getHeaderInfo(CaseData caseData) {
+        StringBuilder headerInfo = new StringBuilder();
+        headerInfo.append("Case Name: " + caseData.getApplicantCaseName());
+        headerInfo.append("\n\n");
+        headerInfo.append(getFamilyManNumber(caseData));
+        headerInfo.append("\n\n");
+        return headerInfo.toString();
+    }
+
+    private String getFamilyManNumber(CaseData caseData) {
+        if (caseData.getFl401FamilymanCaseNumber() == null && caseData.getFamilymanCaseNumber() == null) {
+            return FAMILY_MAN_ID;
+        }
+        return caseData.getCaseTypeOfApplication().equalsIgnoreCase(FL401_CASE_TYPE)
```

Comment:
can we change the comparison parties. constant should be first parameter.

Reply:
updated as instructed